packages/rn-tester/RCTTest/FBSnapshotTestCase/UIImage+Diff.m

```diff
@@ -21,15 +21,16 @@ - (UIImage *)diffWithImage:(UIImage *)image
   UIGraphicsImageRenderer *const renderer = [[UIGraphicsImageRenderer alloc] initWithSize:imageSize
                                                                                    format:rendererFormat];
 
-  return [renderer imageWithActions:^(UIGraphicsImageRendererContext *_Nonnull context) {
+  return [renderer imageWithActions:^(UIGraphicsImageRendererContext *_Nonnull rendererContext) {
+    CGCotextRef context = rendererContext.CGContext;
```

Comment:
`CGCotextRef` -> `CGContextRef`

Also you can make this a `const`

Reply:
Done! Thanks for the catch